### ISSUE TITLE
Restore on-hover data display?

### ISSUE THREAD

**Issue by rod-glover** [pacificclimate/dash-dv-explorer#77]
When we refactored to perform on-demand data loading, we broke the on-hover data display. The superficial problem is that the on-hover callback fires very quickly while moving the cursor, and causes multiple attempts to load all the necessary data files 36 of them) and extract data from them. 

A somewhat naive fix would be to introduce some kind of signal that a data file is being loaded and filter out hover events accordingly. But this would ignore the fact that the on-hover callback is shared amongst all users (Dash is stateless), and so different users could end up requesting different datasets to be loaded and some kind of thrashing ensue, which would doubtless break the functionality in much the same way.

That said, if "loading" a dataset amounts only to loading its metadata (which it appears it does), then we can actually cache a large number of datasets and not break as above. This is because the on-hover data display only extracts a single cell from each dataset, and so the amount of data actually brought into memory is tiny. The problem of high rate events still must be solved but that may be sufficient.

There may be a much better fix but at the moment I am not sure what it is.

**Comment by rod-glover** [comment]
Rod, what _were_ you trying to say? This doesn't make much sense now. Hover events are rapid, and for each hover event we would have to extract the cell corresponding to the hover position from each dataset. There is no apparent way around this, and the amount of data (basically, all datasets) is not feasible to hold in memory.

The only meaning I can see in the whole discussion above is that maybe we could cache all the metadata for each dataset, and load actual data only when it is needed. That might improve performance in some areas, but it still comes up against:
- lat(rlat, rlon), lon(rlat, rlon) are large "metadata", if that was intended here
- data values are still needed at high rate, and loading them is necessarily slow

In short, I don't think this is a go.

**Comment by rod-glover** [comment]
Closing because completely unnecessary.
